lib/ssl/test/ssl_api_SUITE.erl

```diff
@@ -2822,6 +2828,12 @@ options_verify(Config) ->  %% fail_if_no_peer_cert, verify, verify_fun, partial_
      ?OK(#{fail_if_no_peer_cert := true, verify := verify_peer, verify_fun := undefined, partial_chain := _},
          [{verify, verify_peer}, {cacerts, [Cert]}], server),
 
+    %% Test ssl option handling. Option values are verified by public_key tests
+    CertPolicyOpts = [{policy_set, [?anyPolicy]}, {explicit_policy, false}],
+
+    ?OK(#{cert_policy_opts := CertPolicyOpts}, [{verify, verify_peer}, {cacerts, [Cert]}, {cert_policy_opts, CertPolicyOpts}],
+        client),
+
```

Comment:
Also add error cases below.

Reply:
Do we need more error cases than tested by invalid_options test?

Reply:
I have added error cases, e.g. see line 2845, for all options, so I think it should be at least here, if you want to add it to the (old) separate testcases which only had a few error cases you can do so.

Reply:
In that case I think we should remove the old test case and add the error test here.